buddyboss/buddyboss-platform
# HTML tags are showing in the forums when subscribers post copy-paste content from an external DOCS to forums

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1777 

### How to test the changes in this Pull Request:

1. Go to a Group that has forum activated
2. Create a new discussion as a Subscriber.
3. Copy and paste content from an external doc. (Sample doc attached).
4. See error.

### Proof Screenshots or Video
http://prntscr.com/vm0jdz



### Other information:
